workflows/metatdenovo.nf

```diff
@@ -11,9 +11,10 @@ WorkflowMetatdenovo.initialise(params, log)
 
 // Validate parameters Orfcaller:
 def valid_params = [
-    orf_caller  : ['prodigal']
+    orf_caller  : ['prodigal','prokka']
 ]
 ORF_CALLER_PRODIGAL = 'prodigal'
+ORF_CALLER_PROKKA   = 'prokka'
```

Comment:
If we use constants, like these, they should be used in the definition of `valid_params` too, i.e.:
```
def valid_params = [
    orf_caller = [ORF_CALLER_PRODIGAL, ORF_CALLER_PROKKA]
]
```
(The definitions of the constants need to come before this then.)
I'm not sure this is common practice in nf-core.

Reply:
I haven't found any examples in the nf-core pipelines I've looked at, so thanks for the comment, will fix.